.github/workflows/test.yml

```diff
@@ -242,7 +242,7 @@ jobs:
             --packages-up-to-regex amlip \
             --event-handlers=console_direct+ \
             --metas src/AML-IP/.github/workflows/test_colcon.meta \
-            --mixin coverage-gcc asan-gcc
+            --mixin coverage-gcc
```

Comment:
Actually flaky tests do not need coverage as we do not merge coverages from jobs (yet)